src/main/scala/edu/ie3/osmogrid/io/input/pbf/PbfWorker.scala

```diff
@@ -0,0 +1,86 @@
+/*
+ * © 2021. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.osmogrid.io.input.pbf
+
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.typed.{ActorRef, Behavior, PostStop}
+import com.acervera.osm4scala.EntityIterator.fromBlob
+import com.acervera.osm4scala.model.RelationMemberEntityTypes.RelationMemberEntityTypes
+import com.acervera.osm4scala.model.{
+  Info,
+  NodeEntity,
+  RelationEntity,
+  RelationMemberEntity,
+  WayEntity
+}
+import edu.ie3.osmogrid.io.input.pbf.PbfGuardian.stopAndCleanup
+import edu.ie3.osmogrid.model.Osm4ScalaMapper.{osmNode, osmRelation, osmWay}
+import edu.ie3.osmogrid.model.OsmoGridModel.LvOsmoGridModel
+import edu.ie3.osmogrid.model.{OsmoGridModel, SourceFilter}
+import edu.ie3.util.geo.GeoUtils
+import edu.ie3.util.osm.model.OsmContainer
+import edu.ie3.util.osm.model.OsmContainer.ParOsmContainer
+import edu.ie3.util.osm.model.OsmEntity.{MetaInformation, Node, Relation, Way}
+import edu.ie3.util.osm.model.OsmEntity.Relation.{
+  RelationMember,
+  RelationMemberType
+}
+import org.openstreetmap.osmosis.osmbinary.fileformat.{Blob, BlobHeader}
+
+import java.time.ZonedDateTime
+import java.util.UUID
+import scala.collection.parallel.immutable.ParSeq
+
+private[pbf] object PbfWorker {
+
+  // external request protocol
+  sealed trait PbfWorkerMsg
+
+  final case class ReadBlobMsg(
```

Comment:
If we'd like to go with `Request` / `Response`, then a `Request` would be missing.